Resources/Audio/_RMC14/Voice/Felinid/attributions.yml

```
@@ -0,0 +1,24 @@
+- files: ["cat_hiss1.ogg", "cat_hiss2.ogg"]
+  license: "CC-BY-4.0"
+  copyright: "Original sound by https://freesound.org/people/secondbody/ - cut out two clips of cat hissing, cleaned up, and converted to ogg."
+  source: "https://freesound.org/people/secondbody/sounds/50357/"
+
+- files: ["cat_meow1.ogg", "cat_meow2.ogg", "cat_meow3.ogg"]
+  license: "CC-BY-3.0"
+  copyright: "Original sound by https://freesound.org/people/ignotus/ - cut out three clips of cat meowing, cleaned up, and converted to ogg."
+  source: "https://freesound.org/people/ignotus/sounds/26104/"
```

This seems to be CC-BY-4.0, not 3.0 https://freesound.org/people/ignotus/sounds/26104/
```suggestion
  license: "CC-BY-4.0"
  copyright: "Original sound by https://freesound.org/people/ignotus/ - cut out three clips of cat meowing, cleaned up, and converted to ogg."
  source: "https://freesound.org/people/ignotus/sounds/26104/"
```